packages/php-wasm/compile/xdebug/Dockerfile

```
@@ -95,7 +101,9 @@ RUN source /root/emsdk/emsdk_env.sh && \
 # Build Xdebug
 RUN set -euxo pipefail; \
 
-	if [[ "${PHP_VERSION:0:1}" -ge 8 ]]; then \
+	if [[ "${PHP_VERSION:0:1}" -ge "8" && "${PHP_VERSION:2:1}" -ge "5" ]]; then \
+		export XDEBUG_BRANCH="master"; \
```

TODO change this to. `xdebug_3_5` once it's released.
For this PR we could use [3.5.0alpha3](https://github.com/xdebug/xdebug/releases/tag/3.5.0alpha3).

```suggestion
	if [[ "${PHP_VERSION:0:1}" -ge "8" && "${PHP_VERSION:2:1}" -ge "5" ]]; then \
		export XDEBUG_BRANCH="xdebug_3_5"; \
```

---

@adamziel would you recommend we use the [3.5.0alpha3](https://github.com/xdebug/xdebug/releases/tag/3.5.0alpha3) version of Xdebug here, or wait with PHP 8.5 support until Xdebug releases 3.5?

I'm thinking of shipping it with 3.5.0alpha3 if everything works correctly and recompile only Xdebug once 3.5 is released.

---

[I updated the version to 3.5.0alpha3 and recompiled](https://github.com/WordPress/wordpress-playground/pull/2950/commits/4d64b75986b1660d2635b34669a16003a30d78b5)

---

alpha sounds good! thanks